client/src/components/Home.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Link } from "react-router-dom";
```

Comment:
I noticed in both the command terminal as well as the browser console that a few minor warning reminders were displayed about unused variables. In VS Code itself this is signaled by variable names being faded out or dark, for instance the Link import on line 2 here. It's always good practice to go through and clean up things that ended up not being used.